nautobot/core/settings.py

```diff
@@ -212,6 +212,13 @@
 # Nautobot related - May be overridden if using custom social auth backend
 SOCIAL_AUTH_BACKEND_PREFIX = "social_core.backends"
 
+# OAuth2/OIDC claim where the list of groups the authenticating user is a part of
+SSO_CLAIMS_GROUP = os.getenv("NAUTOBOT_SSO_CLAIMS_GROUP", "groups")
+# list of groups that an authenticating user can be a part of to get the Django staff permission
+SSO_STAFF_GROUPS = os.getenv("NAUTOBOT_SSO_STAFF_GROUPS", "").split(",")
+# list of groups that an authenticating user can be a part of to be a Django super user
+SSO_SUPERUSER_GROUPS = os.getenv("NAUTOBOT_SSO_SUPERUSER_GROUPS", "").split(_CONFIG_SETTING_SEPARATOR)
```

Comment:
Any reason for using `_CONFIG_SETTING_SEPARATOR` on one but not both of these?

Reply:
Too many irons in too many fires. I missed it.